tests/Agent/IntegrationTests/UnboundedIntegrationTests/NServiceBus/NsbAsyncCmdHandlerTests.cs

```diff
@@ -0,0 +1,153 @@
+﻿// Copyright 2020 New Relic, Inc. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+
+using MultiFunctionApplicationHelpers;
+using NewRelic.Agent.IntegrationTestHelpers;
+using NewRelic.Testing.Assertions;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace NewRelic.Agent.UnboundedIntegrationTests.NServiceBus
+{
+    public abstract class NsbAsyncCmdHandlerTests<TFixture> : NewRelicIntegrationTest<TFixture>
+        where TFixture : ConsoleDynamicMethodFixture
+    {
+        private readonly ConsoleDynamicMethodFixture _fixture;
+
+        protected NsbAsyncCmdHandlerTests(TFixture fixture, ITestOutputHelper output) : base(fixture)
+        {
+            _fixture = fixture;
+            _fixture.TestLogger = output;
+            _fixture.SetTimeout(TimeSpan.FromMinutes(3));
+
+            _fixture.AddCommand("NServiceBusDriver StartNServiceBusWithAsyncCommandHandler");
+            _fixture.AddCommand("NServiceBusDriver SendCommand");
+            _fixture.AddCommand("RootCommands DelaySeconds 5");
+            _fixture.AddCommand("NServiceBusDriver StopNServiceBus");
+
+            _fixture.Actions
+            (
+                setupConfiguration: () =>
+                {
+                    var configPath = fixture.DestinationNewRelicConfigFilePath;
+                    var configModifier = new NewRelicConfigModifier(configPath);
+                    configModifier.ForceTransactionTraces();
+
+                    CommonUtils.ModifyOrCreateXmlAttributeInNewRelicConfig(configPath, new[] { "configuration", "transactionTracer" }, "explainEnabled", "true");
```

Comment:
Since this is not a datastore test, why are we enabling explain plans?

Reply:
Copy/pasted too much from the cosmos tests (whoops).